### DISCUSSION TITLE
Does BoilR run the original launcher in background?

### DISCUSSION THREAD

**Discussion by Siegfriedmk** [PhilipK/BoilR#362] [Q&A]
Hello. I'm using BoilR via moraroy's NonSteamLauncher app.
It works fine, but sorry if I'm dumb, I'm really curious about this thing: does BoilR also run the launcher in the background, when you run a game via Gaming mode on Steam Deck?
Practical example: on Steam Deck, I installed Death Stranding on Epic Launcher, then imported it in Steam Gaming Mode with BoilR. When I run the game from the Gaming mode, there are no evidences it also run the Epic Launcher in background...no splash screens or loading screens.
Does it run or not? You know...I need it for achievements and cloud saves.

**Comment by PhilipK** [comment]
You can choose in settings, if it should run the launcher or not. Some
games prefer running the launcher others do not, can set the default and
you can set individual game preferences.

**Reply by Siegfriedmk** [reply]
Thanks, it worked flawlessly. Can you do this also for Uplay/Ubi Connect games? I don't see it in the list present in Settings menu.

**Comment by Siegfriedmk** [comment]
Ok, I'll answer myself. BoilR don't see games installed on Uplay/UConnect. Instead, it seems to have a predetermined list with some Ubisoft games, like Ghost Recon Wildlands. Games I don't even own.
